old/src/lib/navigation.js

```diff
@@ -383,7 +406,8 @@ const injectFunction = async (sleepTime, shouldCheckLoad) => {
 
       const maxDynamicWaits = 1;
       let i;
-      for (i = 0; shouldCheckLoad & (i < maxDynamicWaits); i++) {
+      for (i = 0; shouldCheckLoad && i < maxDynamicWaits; i++) {
+        /*
```

Comment:
I think it's better to do `//` comments when commeneting stuff out, so the whole commented section is in the diff / change history, and you can see it's commented without needing syntax highlighting